themes/godotengine/layouts/features.htm

```diff
@@ -183,13 +200,13 @@ <h2>Multi-platform editor</h2>
       <h2>Multi-platform deploy</h2>
       <em>Deploy games everywhere!</em>
       <ul>
-        <li><strong>Desktop platforms:</strong> Windows, macOS, Linux, UWP, *BSD</li>
-        <li><strong>Mobile platforms:</strong> iOS, Android</li>
+        <li><strong>Export to desktop platforms:</strong> Windows, macOS, Linux, and *BSD.</li>
```

Comment:
UWP is still supported. It's not necessarily battle tested/usable in production but some users seem to get it to work (and we've seen some Xbox ports using it already), so I'd keep it mentioned.